hardware_enablement/kmm-release-notes.adoc

```diff
@@ -28,4 +27,297 @@ toc::[]
+
+** *Result:* The more accurate timestamps enable the Kernel Module Management (KMM) Operator to reload the kernel module after the node reboot sequence.
+
+// MGMT-20775
+* Redeploying a module that uses in-cluster builds fails with the `ImagePullBackOff` policy.
+
+** *Cause*: On the Kernel Module Management (KMM) Operator, the `pullPolicy` for the puller pod and the worker pod is different.
+
+** *Consequence*: An image can be considered as existing while it fact it is not.
+
+** *Fix*: Make the pull policy of the pull pod the same as the pull policy defined in the KMM Module, as this is what is used by the worker pod.
+
+** *Result:* The MIC will represent the state of the image in the same way the worker pod will be able to access it.
+
+// MGMT-20785
+* The MIC controller creates two pull-pods when it should create just one.
+
+** *Cause*: On the Kernel Module Management (KMM) Operator, the `ModuleImagesConfig` (MIC) controller may create multiple pull-pods for the same image.
+
+** *Consequence*: Resources are wasted and confusion is caused.
+
+** *Fix*: The `CreateOrPatch` MIC API receives a slice of `ImageSpecs`, as the input is created by going over the the target nodes and adding their images to the slice, so any duplicate `ImageSpecs`, are now filtered out.
+
+** *Result:* The KMM Operator works as expected.
+
+// MGMT-20827
+* The `job.dcDelay` example in the documentation should specify `0s` instead of 0.
+
+** *Cause*: The Kernel Module Management (KMM) Operator default `job.gcDelay` duration field is 0s but the documentation says it is `0`.
+
+** *Consequence*: This can lead a customer in the wrong direction setting a custom value of 60 instead of 60s or 1m that will resolve in an error due to the wrong input type.
+
+** *Fix*: The `job.gcDelay` in the documentation is updated to have a default value of 0s.
+
+** *Result:* Users are less likely to get confused.
+
+// MGMT-20835
+* The KMM Operator Hub environment does not work because of missing MIC and MBSC CRDs.
+
+** *Cause*: The Kernel Module Management (KMM) Operator Hub environment only generates Custom Resource Definitions (CRD) files based on the 'api-hub/' directory. As a result, this does not contain some CRDs required for the KMM Operator Hub environment, such as, 'ModuleImagesConfig' (MIC) resource and Managed Kubernetes Service (MBSC).
+
+** *Consequence*: The KMM Operator Hub environment can not work because it tries to start controllers reconciling CRDs that do not exist in the cluster.
+
+** *Fix*: The fix generates all CRD files into the 'config/crd-hub/bases' directory, but only applies the resources to the cluster that it actually needs.
+
+** *Result:* The KMM Operator Hub environment works as expected.
+
+// MGMT-20852
+* The KMM Operator Hub environment cannot build when finalisers can not be set on a resource.
```

Comment:
Suggestion based on the assumtion that this refers to [OperatorHub](https://operatorhub.io/about) and not the hub-spoke of KMM
```suggestion
* The KMM OperatorHub environment cannot build when finalisers are not set on a resource.
```